modules/update-other-images.adoc

```diff
@@ -48,8 +37,24 @@ $ oc -n stackrox set image ds/collector compliance=registry.redhat.io/rh-acs/mai
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/rh-acs/collector:{collector-version}
 ----
+. Apply the patch for Sensor:
++
+[WARNING]
+====
+Applying the patch for Sensor is only required when you are upgrading from {product-title} version between 3.0.44 and 3.0.54. Otherwise, skip this instruction.
+====
++
+[source,terminal]
+----
+$ oc -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","env":[{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"fieldPath":"metadata.namespace"}}}],"volumeMounts":[{"name":"cache","mountPath":"/var/cache/stackrox"}]}],"volumes":[{"name":"cache","emptyDir":{}}]}}}}'
+----
 . Apply the following cluster role and cluster role binding:
 +
+[WARNING]
+====
+Applying the cluster role and the cluster role binding is only required when you are upgrading from {product-title} version between 3.0.44 and 3.0.54. Otherwise, skip this instruction.
```

Comment:
```suggestion
Applying the cluster role and the cluster role binding is only required when you are upgrading from {product-title} version between 3.0.44 and 3.0.54. Otherwise, skip this step.
```